main.py

```diff
@@ -1,6 +1,7 @@
 import tkinter as tk
 from tkinter import filedialog
 from tkinter import messagebox
+import signal
```

Comment:
Correct me if I'm wrong but I don't think we need that import

Reply:
No you are right